rs/nns/governance/canister/canister.rs

```diff
@@ -134,27 +135,20 @@ are best practices for making use of the unsafe block:
 /// This should only be called once the global state has been initialized, which
 /// happens in `canister_init` or `canister_post_upgrade`.
 fn governance() -> &'static Governance {
-    unsafe { GOVERNANCE.as_ref().expect("Canister not initialized!") }
+    unsafe { &*GOVERNANCE.with(|g| g.as_ptr()) }
```

Comment:
I agree. That's the main reason I wrote the previous version. It's just that the only way it can be achieved is to use `Option<Governance>` for the initial state, but it makes the global state a bit cumbersome to use when accessing it through `LocalRef::with`